app/helpers/tolk/application_helper.rb

```diff
@@ -17,7 +17,7 @@ def highligh_interpolations(value)
     end
 
     def format_i18n_text_area_value(value)
-      yaml_value(value).to_s.force_encoding("UTF-8")
+      value = yaml_value(value).to_s.dup.force_encoding("UTF-8")
```

Comment:
```suggestion
      yaml_value(value).to_s.dup.force_encoding("UTF-8")
```
C'est utile le `value =`?